Sources/_InternalTestSupport/SkippedTestSupport.swift

```diff
@@ -0,0 +1,70 @@
+
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import class Foundation.FileManager
+import class Foundation.ProcessInfo
+import Testing
+
+extension Trait where Self == Testing.ConditionTrait {
+    /// Skip test if the host operating system does not match the running OS.
+    public static func requireHostOS(_ os: OperatingSystem, when condition: Bool = true) -> Self {
```

Comment:
Is this used anywhere outside of this package? In general we try to stick to more restrictive `package` if there's a need to use APIs in other modules of this same package. Otherwise there's a risk of accidentally and unnecessarily creating a public API that other packages may have unintended and undesirable dependencies.

```suggestion
    package static func requireHostOS(_ os: OperatingSystem, when condition: Bool = true) -> Self {
```

Reply:
This is currently a helper.  Nothing is using it at the moment, but the plan it to has Swift Testing tests use this traits when a test requires a specific host OS.